met-web/src/components/engagement/new/create/index.tsx

```diff
@@ -1,51 +1,202 @@
+        defaultValues: {
+            name: '',
+            start_date: undefined,
+            end_date: undefined,
+            is_internal: undefined,
+            slug: '',
+        },
+        mode: 'onBlur',
+    });
+
+    const onSubmit = async (data: EngagementCreationData) => {
+        fetcher.submit(
+            {
+                name: data.name,
+                start_date: data.start_date.format('YYYY-MM-DD'),
+                end_date: data.end_date.format('YYYY-MM-DD'),
+                is_internal: data.is_internal,
+                slug: data.slug,
+            },
+            {
+                method: 'post',
+                action: '/engagements/create/',
+            },
+        );
+    };
+
+    const {
+        control,
+        handleSubmit,
+        formState: { errors, isDirty, isValid, isSubmitting },
+    } = engagementCreationForm;
+
     return (
         <ResponsiveContainer>
+            <AutoBreadcrumbs />
+            <FormProvider {...engagementCreationForm}>
+                <Form onSubmit={handleSubmit(onSubmit)}>
+                    <Header1 sx={{ mb: 0 }}>New Engagement</Header1>
+                    <Header2 weight="thin">Create a new engagement in six easy configuration steps.</Header2>
+                    <BodyText>
+                        <FontAwesomeIcon icon={faInfoCircle} style={{ marginRight: '8px' }} />
+                        <i>
+                            You will be able to modify the configuration of your engagement later in the case the
+                            parameters of your engagement change.
+                        </i>
+                    </BodyText>
+                    <br />
+                    <Header2 decorated>Configure Engagement</Header2>
+                    <BodyText bold size="large">
+                        In order to create your new engagement we’ll need to configure a few things first.
+                    </BodyText>
+                    <br />
+                    <DetailsContainer sx={{ maxWidth: '1000px', margin: { xs: '0 -16px', sm: '0' } }}>
+                        <Detail>
+                            <Controller
+                                control={control}
+                                name="name"
+                                rules={{ required: 'Engagement title is required' }}
+                                render={({ field }) => (
+                                    <TextField
+                                        {...field}
+                                        error={errors.name?.message}
+                                        counter
+                                        maxLength={50}
+                                        title="1. What is the title of your Engagement?"
+                                        instructions="Titles should succinctly describe what your engagement is about in 60 characters or less."
+                                        placeholder="Engagement title"
```

Comment:
Many screen readers do not read placeholders and some may not properly interpret the title attribute on a form field. We should make sure to include a label so the field is properly identified by them.

```suggestion
                                        placeholder="Engagement title"
                                        label="Engagement title"
```